rest_framework/serializers.py

```diff
@@ -1477,7 +1477,7 @@ def get_validators(self):
         # If the validators have been declared explicitly then use that.
         validators = getattr(getattr(self, 'Meta', None), 'validators', None)
         if validators is not None:
-            return validators[:]
+            return list(validators)
```

Comment:
Hm. We should *probably* also have a test for serializer validators too.

Reply:
good idea; done